applications/FluidDynamicsApplication/tests/auxiliary_files/cube_few_elements.mdpa

```diff
@@ -0,0 +1,140 @@
+Begin Properties 1
```

Comment:
I'd rename this one to something like `cube_tetrahedra_elements_coarse`.

Reply:
done